### PR TITLE
DOC example for data_per_streamline usage

### PR DESCRIPTION
**Pull request by lb-97**
Edited doc page for issue #2735

### CODE REVIEW

**Comment by lb-97** [comment]
Hi @skoudoro,
Hope you're doing good. There are a couple of checks that failed during testing that seem to not be related to my commit. Is there a way to re-run workflows? Thank you.

**Comment by Garyfallidis** [comment]
The approach seems incorrect. I do not think that ``data_per_streamline`` expects a dictionary as input.

File: doc/examples/streamline_formats.py [hunk 1]
```diff
@@ -145,6 +145,17 @@
...
 save_tractogram(rpt_sft, 'rpt.trk')
 
+"""
+Any StatefulTractogram instance consists of clusters of streamlines. Cluster 
+based information such as number of cluster label, streamlines within each cluster
+can be added through a setter function.
+"""
+
+data = {}
+for i in range(len(cc_sft.streamlines)):
```
Comment (skoudoro) [inline doc/examples/streamline_formats.py:155]: to be more pythonic please use `enumerate` instead `range(len(`

Comment (lb-97) [reply doc/examples/streamline_formats.py]: Done

**Comment by skoudoro** [comment]
Thank you @lb-97 .

I will close this PR for now because the information is incomplete and misleading.  it will have more value with a concrete example (using Quickbundle) 

Feel free to look more in details at Quickbundle and do not hesitate to reopen this PR.

Thanks!

**Pull request closed without merging**